r/adbcdrivermanager/bootstrap.R

```diff
@@ -19,8 +19,10 @@
 # implementation from the repo. We also run this from configure, so do nothing
 # if we aren't sitting within the repo (e.g., installing a source package from a
 # tarball).
+dir.create("src/arrow-adbc", showWarnings=FALSE)
+file.copy("../../c/include/arrow-adbc/adbc.h", "src/arrow-adbc/adbc.h")
+
 files_to_vendor <- c(
-  "../../adbc.h",
   "../../c/driver_manager/adbc_driver_manager.h",
```

Comment:
Maybe should also install the adbc_driver_manager.h header into the arrow-adbc subdirectory? Seems like the only time it is included here is by adbc_driver_manager.cc so still "works" as is, but in turn this is a bit inconsistent with how we install that header

Reply:
Yes! That would be much appreciated